src/pages/home/report/ContextMenu/ReportActionContextMenu.ts

```diff
@@ -64,8 +64,7 @@ function hideContextMenu(shouldDelay?: boolean, onHideCallback = () => {}) {
         return;
     }
     if (!shouldDelay) {
-        contextMenuRef.current.hideContextMenu(onHideCallback);
-
+        onHideCallback();
```

Comment:
This change causes the context menu to stay open on mobile.


https://github.com/user-attachments/assets/43b33224-f588-4374-90f5-db15b7458f74

Reply:
Thanks @j-piasecki for pointing this out. 
@teneeto Let us raise another PR to fix this. I think we can revert this line to the original as it was not required to fix the issue at hand here. What do you think?